talend-component-maven-plugin/src/it/web/.jenkins/scripts/server-registry-start.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+#
+#  Copyright (C) 2006-2022 Talend Inc. - www.talend.com
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# set -xe
+
+# Start a server in registry mode
+# Parameters:
+[ -z ${1+x} ] && usage "Parameter 'install_dir'"
+[ -z ${2+x} ] && usage "Parameter 'coverage_dir'"
+[ -z ${3+x} ] && printf "Parameter 'server_port' use the default value: 8080"
+
+INSTALL_DIR=${1}
+COVERAGE_DIR=${2}
+PORT=${3:-"8080"}
+
+# Constants
+EXTRA_INSTRUMENTED="vault-client"
+DISTRIBUTION_DIR="${INSTALL_DIR}/component-server-distribution"
+LIB_DIR="${DISTRIBUTION_DIR}/lib"
+LIB_BACKUP_DIR="${COVERAGE_DIR}/lib_backup"
+LIB_INSTRUMENTED_DIR="${COVERAGE_DIR}/lib_instrumented"
+SOURCES_DIR="${COVERAGE_DIR}/src"
+JACOCO_CLI_PATH="${LIB_DIR}/jacococli.jar"
+MEECROWAVE_LOG_PATH="${DISTRIBUTION_DIR}/logs/meecrowave.out"
```

Comment:
Please use lower case variables inside the script to use local variables instead of environment variables. Unless it is required ?
Same remark for other variables used afterwards.